.travis.yml

```diff
@@ -27,17 +27,21 @@ addons:
     - xvfb
 
 before_install:
+  - export PATH="$PATH:$HOME/.cargo/bin"
   - sudo -E ./.ci/install_cuda.sh
   - sudo -E ./.ci/travis_build_opencv.sh
   - export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib
+  - which rustfmt || cargo install rustfmt-nightly
 
 script:
   - cargo build
   - cargo build --no-default-features
   - cargo test --no-default-features
   - cargo build --features gpu
   - cargo doc --features gpu --no-deps
-  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then cargo bench ; fi
+  - find src/*.rs -exec rustfmt {} \;
+  # Available until `cargo fmt` landed in stable rust 1.24
```

Comment:
As I said, you don't need `cargo fmt` here, just run `rustfmt` manually from `.cargo` directory.